core/invoker/src/main/resources/application.conf

```diff
@@ -10,6 +10,14 @@ whisk {
     poll-interval: 5 minutes
   }
 
+  # Docker < 1.13.1 has a known problem: if more than 10 containers are created (docker run)
+  # concurrently, there is a good chance that some of them will fail.
+  # See https://github.com/moby/moby/issues/29369
+  # Use a semaphore to make sure that at most 10 `docker run` commands are active
+  # the same time.
+  # 0 means that there are infinite parallel runs.
+  docker.parallel-runs: 10
```

Comment:
Should we refactor the config a bit? I'm envisioning something like:

```
docker.client {
  parallel-runs: 10,
  timeouts {
    ..
  }
}
```

Which would change the code to:

```scala
case class DockerClientConfig(parallelRuns: Int, timeouts: DockerClientTimeoutConfig)
```

And makes it a bit more clean in general. WDYT?